controllers/host/host_controller.go

```diff
@@ -1602,36 +1603,7 @@ func (r *HostReconciler) ReconcileExistingHost(client *gophercloud.ServiceClient
 		return err
 	}
 
-	// Check factory install config map
-	logHost.V(2).Info("checking factory install config map", "namespace", reqNs)
-	factory, err := r.CloudManager.GetFactoryInstall(reqNs)
-	if err != nil {
-		return perrors.Wrap(err, "failed to get factory install config map")
-	}
-
-	if factory && host.IsUnlockedEnabled() {
```

Comment:
Can you help restore this logic? It was observed that the unlock action is very fast to kill the dm pod which causing the final state not updated